src/API/scenarios/scenario.model.js

```diff
@@ -0,0 +1,66 @@
+import Promise from 'bluebird';
```

Comment:
we promisify mongoose with bluebird. Bluebird promises are used here to keep promise API the same (for example [.error ](http://bluebirdjs.com/docs/api/error.html#error) method), so we can always expect a bluebird promise from db operations.